pkg/util/mount/mount_windows.go

```diff
@@ -0,0 +1,131 @@
+// +build windows
+
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mount
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+type Mounter struct {
+	mounterPath string
+}
+
+func (mounter *Mounter) Mount(source string, target string, fstype string, options []string) error {
+	if source == "tmpfs" || len(target) < 3 {
+		glog.Infof("azureMount: Skip mounting source (%q), target (%q)\n, with arguments (%q) in windows", source, target, options)
+		return nil
+	}
+
+	if !strings.HasPrefix(target, "c:") && !strings.HasPrefix(target, "C:") {
+		target = "c:" + target
+	}
+	parentDir := filepath.Dir(target)
+	err := os.MkdirAll(parentDir, 0755)
+	if err != nil {
+		glog.Infof("mkdir(%q) failed: %v\n", parentDir, err)
+		return err
+	}
+
+	if len(options) != 1 {
+		glog.Warningf("azureMount: mount options(%q) command(%n) not equal to 1, skip mounting.\n", options, len(options))
+		return nil
+	}
+	cmd := options[0]
+
+	driverLetter, err := getAvailabeDriverLetter()
+	if err != nil {
+		return err
+	}
+	driverPath := driverLetter + ":"
+	cmd += fmt.Sprintf(";New-SmbGlobalMapping -LocalPath %s -RemotePath %s -Credential $Credential", driverPath, source)
+
+	_, err = exec.Command("powershell", "/c", cmd).CombinedOutput()
```

Comment:
there are only two powershell command invocation now, I will add if there is more